<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarDlgClass</class>
 <widget class="QDialog" name="ProgressBarDlgClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>87</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>199</width>
    <height>87</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>199</width>
    <height>87</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>199</width>
    <height>87</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Wait...</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
    <item>
      <widget class="QProgressBar" name="main_progressBar">
        <property name="value">
          <number>24</number>
        </property>
      </widget>
    </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgressBarDlgClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
